### PR TITLE
Api 12391 update db

### PR DESCRIPTION
**Pull request by FonzMP**
reworks api <- api_environment -> environment layout. some spec and factorybot changes as well.
Also reintroduces brakeman 👋

### CODE REVIEW

File: app/services/user_service.rb [hunk 1]
```diff
@@ -46,6 +43,17 @@ def create_or_update_consumer(user, params)
 
   private
 
+  def add_api_to_consumer(api_name, consumer)
+    api_id = ApiRef.find_by(name: api_name.strip)[:api_id]
+    api_model = Api.find(api_id)
+    env = Environment.find_by(name: 'sandbox')
```
Comment (SlakrHakr) [inline app/services/user_service.rb:49]: this should probably be hardcoded at the controller level, then passed down into the service as just an environment, so we can use this same service for giving production access in the future

File: spec/api/v0/consumers_spec.rb [hunk 2]
```diff
@@ -40,60 +44,23 @@
...
-          description: 'Signing up for Patti',
-          email: 'doug@douglas.funnie.org',
-          firstName: 'Douglas',
-          lastName: 'Funnie',
-          oAuthApplicationType: 'web',
-          oAuthRedirectURI: 'http://localhost:3000/callback',
-          organization: 'Doug',
-          termsOfService: true
-        }
-      end
```
Comment (SlakrHakr) [inline spec/api/v0/consumers_spec.rb:95]: 💯

File: spec/factories/api_refs.rb [hunk 3]
```diff
@@ -2,6 +2,6 @@
 
 FactoryBot.define do
   factory :api_ref do
-    name { 'MyString' }
+    name { Faker::Hipster.word }
```
Comment (SlakrHakr) [inline spec/factories/api_refs.rb:5]: https://c.tenor.com/390VTlJCoWYAAAAC/tenor.gif

**Review by SlakrHakr — APPROVED** [review]

File: app/services/user_service.rb (hunk 1 above)
Comment (FonzMP) [reply app/services/user_service.rb]: [79e0bd7](https://github.com/department-of-veterans-affairs/lighthouse-platform-backend/pull/57/commits/79e0bd7913b4aadc163fa0e64c73788e14ed4d49)

**Review by SlakrHakr — APPROVED** [review]

**Pull request merged**